src/components/Player.js

```diff
@@ -19,12 +19,41 @@ const Player = ({currentSong, isPlaying, setIsPlaying}) => {
       setIsPlaying(!isPlaying);
     }
   };
+  //function used to extract currentTime of song and duration
```

Comment:
with these comments its a bit obvious that the following line is a function. simplify your efforts and just start after 'used to-'